docs/en/changelogs/2.2.0.md

```diff
@@ -142,9 +142,9 @@ Sitemap.php
       * Translatable, for multilingual content
   * New third party libraries
       * PHP OpenID (http://openidenabled.com/php-openid/)
-      * Browscap (http://garetjax.info/projects/browscap/)
+      * Browscap (http://browscap.org/)
       * Plotr (http://www.solutoire.com/plotr/)
-      * SWFUpload (http://profandesign.se/swfupload/)
+      * SWFUpload (http://profandesign.se/)
```

Comment:
OK, I'll remove the link :)

If you look at the uat docs test server you will see that all versions below 3.2 are now "archived". So, for versions 3.1 and below, I am not really worrying about the content of the links or finding suitable replacements/updates. I am only disabling the links that are broken by back-ticking them, which disables the link but keeps the information visible. My ultimate goal here is to clean up the .htaccess file and the rewrite rules.

For 3.2 and up, I will pay more attention to what is being linked to and take care to find appropriate updates/replacements of the broken links. If needed, I can go back and do the same thing for <= 3.1 but it seems more important to get >= 3.2 links fixed and pointing to correct material first. @camfindlay do you agree ?